composer.json

```diff
@@ -25,7 +25,7 @@
     "require": {
         "php": ">= 7.1",
         "ext-json": "*",
-        "illuminate/contracts": "5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0",
+        "illuminate/contracts": "5.7.*",
```

Comment:
why constrict to 5.7.x?

Reply:
This PR is not going to land in `master`, it is only meant to find an issue specific to this Laravel version.